queries/php/injections.scm

```diff
@@ -2,6 +2,18 @@
 
 (comment) @phpdoc
 
+(heredoc
+ (heredoc_body) @content
+ (heredoc_end) @language
+ (#set! "language" @language)
+ (#downcase! "language"))
+
+(nowdoc
+ (nowdoc_body) @content
+ (heredoc_end) @language
```

Comment:
why is nowdoc has an ending in the form of heredoc ... that seems unintuitive

Reply:
I assume that's copy-pasta.

Reply:
No copy-pasta, that's what the grammar uses :shrug: :

https://github.com/tree-sitter/tree-sitter-php/blob/ff6a35badb0fe373575196bd728a8a53e9be70bd/grammar.js#L1334-L1350

Reply:
Well that I didn't expect :D